content/contributor/runtime/globals.md

```diff
@@ -0,0 +1,107 @@
+---
+title: "Globals"
+sidebartitle: "Globals"
+url: "/contributor/runtime/globals"
+weight: 2
+---
+
+The scene runtime ensures the existence of certain objects and functions inside the sandboxed Javascript environment, available in the global scope.
+
+Most of them are implementations of familiar browser and/or Node concepts, such as `console` or `setImmediate`.
+
+## Globals
+
+The runtime injects 7 definitions in the scene's global scope:
+
+1. [`console`](#console): a subset of the Node/browser `console` object.
+2. [`module`](#module): a subset of Node's `module` object.
+3. [`exports`](#module): a shortcut to `module.exports`, as in Node.
+4. [`require`](#module): an implementation of Node's `require` for a curated list of modules.
+5. [`setImmediate`](#scheduling): an implementation of Node's `setImmediate`.
+6. [`fetch`](#http): a restricted implementation of the browser `fetch` function.
+7. [`WebSocket`](#http): a restricted implementation of the browser `WebSocket` class.
+
+All of these are defined as read-only properties, so they cannot be reassigned. Some will throw exceptions when used unless certain [permissions]({{< ref "/contributor/content/entity-types/scenes#permissions" >}}) are granted to the scene.
+
+
+## Console {#console}
+
+Scenes have access to a `console` object, much like the one provided by a browser or Node environment, though limited to the some of the methods you would normally find.
+
+```ts
+type Console = {
+  log(...args: any): void
+  info(...args: any): void
+  debug(...args: any): void
+  warning(...args: any): void
+  error(...args: any): void
+}
+```
+
+Just like their standard counterparts, each method takes variable arguments of any type and renders them as human-readable messages. For example, this is valid:
+
+```js
+console.log("The thing just appeared", { thing: "foo" }, [1, 2, 3])
+```
+
+The precise behavior of these methods depends on the provider, but the messages must be accessible to scene developers debugging their code.
+
+{{< info >}}
+In the Foundation's browser-based World Explorer, log messages appear in the developer tools panel.
```

Comment:
Prevent mentioning the foundation, maybe something like `In the browser-based World Explorer, log messages appear in the developer tools panel.`